templates/bot/csharp/command-and-response/Properties/launchSettings.json

```diff
@@ -13,7 +13,7 @@
       "hotReloadProfile": "aspnetcore"
     },
     // Debug project only (without launching Teams)
-    "Start Project (not in Teams)": {
+    "Start (not in Teams)": {
```

Comment:
Also update the description in `GettingStarted.txt`?

Reply:
`GettingStarted` only mentions the Teams one, not this one.